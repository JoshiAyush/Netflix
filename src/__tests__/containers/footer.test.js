import React from "react";
import { render } from "@testing-library/react";

import { Footer } from "../../containers/index.js";

describe("<Footer />", () => {
  it("renders the <Footer /> with populated data", () => {
    const { container, getByText } = render(<Footer />);

    expect(getByText("Questions? Contact us")).toBeTruthy();
    expect(getByText("FAQ")).toBeTruthy();
    expect(getByText("Investor Relations")).toBeTruthy();
    expect(getByText("Ways to Watch")).toBeTruthy();
    expect(getByText("Coporate Information")).toBeTruthy();
    expect(getByText("Netflix Originals")).toBeTruthy();

    expect(container.firstChild).toMatchSnapshot();
  });
});
