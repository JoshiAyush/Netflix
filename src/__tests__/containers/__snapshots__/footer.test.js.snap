// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders the <Footer /> with populated data 1`] = `
<div
  class="style__Container-sc-1qwya3j-0 gROnQU"
>
  <p
    class="style__Title-sc-1qwya3j-4 bNYaSG"
  >
    Questions? Contact us
  </p>
  <p
    class="style__Break-sc-1qwya3j-6 hBxIDB"
  />
  <div
    class="style__Row-sc-1qwya3j-2 jqykhu"
  >
    <div
      class="style__Column-sc-1qwya3j-1 foymxT"
    >
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        FAQ
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Investor Relations
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Ways to Watch
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Coporate Information
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Netflix Originals
      </a>
    </div>
    <div
      class="style__Column-sc-1qwya3j-1 foymxT"
    >
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Help Centre
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Jobs
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Terms of Use
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Contact us
      </a>
    </div>
    <div
      class="style__Column-sc-1qwya3j-1 foymxT"
    >
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Account
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Redeem Gift Cards
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Privacy
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Speed Test
      </a>
    </div>
    <div
      class="style__Column-sc-1qwya3j-1 foymxT"
    >
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Media Centre
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Buy Gift Cards
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Cookie Preferences
      </a>
      <a
        class="style__Link-sc-1qwya3j-3 fqCUMv"
        href="#"
      >
        Legal Notices
      </a>
    </div>
  </div>
  <p
    class="style__Break-sc-1qwya3j-6 hBxIDB"
  />
  <p
    class="style__Text-sc-1qwya3j-5 fSkTJD"
  >
    Netflix United Kingdom
  </p>
</div>
`;
